From 5dffc6e58e439b27cd45f5ebbf720ebfd2f98017 Mon Sep 17 00:00:00 2001
From: Thor aarch64 build <thor-build@localhost>
Date: Tue, 15 Sep 2026 12:50:20 -0400
Subject: [PATCH 2/8] deps/libglu: autoreconf -fi before ./configure (aarch64
 libtool 2.4.x split)

Same libtool-split issue as sqlite; regenerate the autotools files before configure.
---
 build_files/build_environment/cmake/libglu.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build_files/build_environment/cmake/libglu.cmake b/build_files/build_environment/cmake/libglu.cmake
index e804f0e10e0..fb6da5b5eb5 100644
--- a/build_files/build_environment/cmake/libglu.cmake
+++ b/build_files/build_environment/cmake/libglu.cmake
@@ -20,6 +20,7 @@ ExternalProject_Add(external_libglu
 
   CONFIGURE_COMMAND ${CONFIGURE_ENV} &&
     cd ${BUILD_DIR}/libglu/src/external_libglu/ &&
+    autoreconf -fi &&
     ${CONFIGURE_COMMAND_NO_TARGET} --prefix=${LIBDIR}/libglu ${LIBGLU_EXTRA_FLAGS}
 
   BUILD_COMMAND ${CONFIGURE_ENV} &&
-- 
2.43.0

